src/Notifier.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Beaver\Dateval;
+
+class Notifier extends \PHPMailer {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition